lib/board.rb

```diff
@@ -0,0 +1,65 @@
+class Board
```

Comment:
SRP: This code mixes the details of several responsibilities: modelling individual cells, modelling lines and modelling the rules of tic tac toe.